eth_keys/backends/native/main.py

```diff
@@ -26,6 +29,20 @@ def ecdsa_sign(self,
         signature = Signature(vrs=signature_vrs, backend=self)
         return signature
 
+    def ecdsa_sign_non_recoverable(self,
+                                   msg_hash: bytes,
+                                   private_key: PrivateKey) -> NonRecoverableSignature:
+        signature_vrs = ecdsa_raw_sign(msg_hash, private_key.to_bytes())
+        signature_rs = signature_vrs[1:]
```

Comment:
I'd like this to be precise, us knowing that `signature_rs` is exactly two values, which this currently allows for it to be anything from empty to many values.

- `signature_vrs[1], signature_vrs[2]` 
- `signature_v, signature_r = signature_vrs[1:]`